SPECS/perl-HTTP-Cookies/perl-HTTP-Cookies.spec

```diff
@@ -64,7 +91,13 @@ make test
 %{perl_vendorlib}/*
 %{_mandir}/man3/*
 
+%files tests
+%{_libexecdir}/%{name}
+
 %changelog
+* Wed Mar 27 2024 Sam Meluch <sammeluch@microsoft.com> - 6.11-1
+- Upgrade package to version 6.11
```

Comment:
you might add a comment here about adding the new `tests` subpackage

Reply:
Comment added